test/language/expressions/tagged-template/call-expression-context-no-strict.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) Copyright 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 12.3.7
+description: Invocation context for call expressions
+info: >
+    A tagged template is a function call where the arguments of the call are
+    derived from a TemplateLiteral. The actual arguments include a template
+    object and the values produced by evaluating the expressions embedded
+    within the TemplateLiteral.
+flags: [noStrict]
+---*/
+var context = null;
+var fn = function() {
+  return function() {
+    context = this;
+  };
+};
+
+fn()`NoSubstitutionTemplate`;
+
+assert.sameValue(context, this);
+
+fn = function() {
+  return () => { context = this; };
+};
+
+context = null;
+fn()`NoSubstitutionTemplate`;
+
+assert.sameValue(context, this);
```

Comment:
I think I follow you. What about if I used [`fnGlobalObject`](https://github.com/tc39/test262/blob/42d16da90113c4739df3a8161ea898e80df70697/harness/fnGlobalObject.js) here, like this:

```diff
 context = null;
 fn()`NoSubstitutionTemplate`;

-assert.sameValue(context, this);
+assert.sameValue(context, fnGlobalObject());
```